src/parser/parse.y

```diff
@@ -12,9 +12,9 @@
 *************************************************************************
 ** This file contains SQLite's SQL parser.
 **
-** The canonical source code to this file ("parse.y") is a Lemon grammar 
+** The canonical source code to this file ("parse.y") is a Lemon grammar
```

Comment:
You should avoid reformatting this file because it makes difficult to keep in sync with original / official SQLite one...

Reply:
Yes sorry this is unintentional, will remove